pkg/server/api/types.go

```diff
@@ -24,6 +24,20 @@ type Request struct {
 	Env map[string]string `json:"env,omitempty"`
 	// CNI configuration passed via stdin to the CNI plugin
 	Config []byte `json:"config,omitempty"`
+	// Annotation for Delegate request
+	InterfaceAttributes *DelegateInterfaceAttributes `json:"delegateAnnotation,omitempty"`
```

Comment:
Do we want to align the attribute name w/ the JSON parameter name ?

Reply:
Oh, sorry to miss that. changed.